chain-impl-mockchain/src/ledger/mod.rs

```diff
@@ -0,0 +1,9 @@
+pub mod ledger;
+
+pub use ledger::*;
+
+cfg_if! {
+   if #[cfg(test)] {
+        pub mod tests;
```

Comment:
This looks like quite an elaborate way to use the conditional `cfg` attribute on a module :) `cfg_if!` is typically used when there are alternative definitions that otherwise would be tedious.